WordPress/Tests/Theme/DeregisterScriptUnitTest.inc

```diff
@@ -0,0 +1,9 @@
+<?php
+
+wp_enqueue_style( 'style', get_stylesheet_uri() ); // this is ok
+
+wp_enqueue_style( 'slider', get_template_directory_uri() . '/css/slider.css', false, '1.1', 'all' ); // this is ok
+
+wp_enqueue_script( 'script', get_template_directory_uri() . '/js/script.js', array( 'jquery' ), 1.1, true ); // ok
+
+wp_deregister_script( 'jquery' ); // this is bad, themes must not deregister scripts
```

Comment:
The only way you can change the jquery URL is by deregistering. Dequeueing just makes it not load.